src/index.ts

```diff
@@ -176,6 +176,7 @@ export type {
   ResumeToken,
   UpdateDescription
 } from './change_stream';
+export type { GSSAPICanonicalizationValues } from './cmap/auth/gssapi';
```

Comment:
The enum and the type need to both be exported, we've collected the enums we export on line 87

Reply:
Updated.